perlmod/Fink/Bootstrap.pm

```diff
@@ -245,9 +245,12 @@ GCC_MSG
 			"of Mac OS X might work with Fink, but there are no " .
 			"guarantees.");
 		$distribution = "10.13";
-	} elsif ($host =~ /^i386-apple-darwin18\.[0-6]/) {
+	} elsif ($host =~ /^i386-apple-darwin18\.[0-5]/) {
 		&print_breaking("This system is supported and tested.");
 		$distribution = "10.14";
+	} elsif ($host =~ /^i386-apple-darwin18\.[6]/) {
+		&print_breaking("This system is supported and tested.");
+		$distribution = "10.14.5";
 	} elsif ($host =~ /^i386-apple-darwin18\./) {
```

Comment:
Shouldn't line 259 now read `$distribution = "10.14.5";`, to ensure that the newer distribution is used for future updates?

Reply:
Yes, at least until Apple breaks things on us again.

Reply:
fixed in 648179b

Reply:
I am a n00b, so what do I have to do to install fink on fresh mac with 10.14.5 .... cheers